_posts/2019-08-07-vim-basics.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: vim Basics
+date: 2019-08-07 10:28:02
+categories: ["text editor", "vim", "best text editor"]
```

Comment:
To make categories a bit more generic (we want other articles to fall into these categories too):

```suggestion
categories: ["learning", "vim"]
```